python/ray/tune/hyperband.py

```diff
@@ -32,24 +32,29 @@ class HyperBandScheduler(FIFOScheduler):
     and band and will spill over to new brackets/bands accordingly.
     """
 
-    def __init__(self, max_iter=200, eta=3):
+    def __init__(self, max_iter=81, eta=None):
         """
         args:
-            max_iter (int): maximum iterations per configuration
-            eta (int): # defines downsampling rate (default=3)
+            max_iter (int): maximum iterations per configuration. Altering
+                this with `eta=None` will not change the bracket count
+                nor size. If eta is changed to an int, then all will change.
```

Comment:
We should also support different reward attributes.

Reply:
Addressed all comments; the PR has now substantiative changes.